src/main/java/com/jamierf/dropwizard/logging/gelf/GelfAppenderFactory.java

```diff
@@ -79,6 +121,12 @@ public void setServer(HostAndPort server) {
         }
 
         appender.getConfig().getDefaultFields().put(GelfTargetConfig.FIELD_FACILITY, applicationName);
+        appender.getConfig().getDefaultFields().putAll(defaultFields);
+        if(!additionalFields.isEmpty())
+        {
+            appender.getConfig().getAdditionalFields().clear();
```

Comment:
Why clear this before adding to it? It seems unnecessary in the current usage, and has potential to remove fields that were added purposely in the future.